### ISSUE TITLE
ES5 support still necessary in some cases

### ISSUE THREAD

**Issue by cmartinez84** [francoisgeorgy/svg-knob#1]
I've really enjoyed the svg-knob in development, but I am using create-react-app, and it doesn't allow module transpiling. Future support would be appreciated!

**Comment by francoisgeorgy** [comment]
Hi Chris,
Thanks for your message. Yeah, I know, I took a bit shortcut by leaving ES5 on the side. I had not much time and I needed to develop something fast for use within my Bass Station web interface. 
I'm doing some React development too and I plan to use my svg-knob and svg-slider in React. I, for sure, will do what is needed to support ES5. I hope to do it in the coming weeks. Depend on my free time. Having a family does not leave much time.
Feel free to fork and submit PR if you want. Help is always welcome.
Best regards
